packages/conf-pic-switch/conf-pic-switch.0.1/opam

```diff
@@ -5,7 +5,7 @@ homepage: "https://github.com/ocaml/opam-repository"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 build: [
-  ["./check.sh"]
+  ["/bin/bash" "./check.sh"]
```

Comment:
this is not a portable invocation of bash unfortunately -- it will be in /usr/local in BSDs. So just `bash` should be sufficient as it will be in the parent PATH